packages/itwinui-css/src/slider/slider.scss

```diff
@@ -18,14 +17,19 @@ $tick-height: $iui-baseline; // 11px
     margin-top: $iui-xxs;
 
     svg {
-      @include iui-icons-default;
+      @include iui-icon-style('m');
+      @include iui-icon-color;
       margin-top: $iui-xs;
     }
   }
 
   .iui-slider-min {
     margin-right: $iui-sm;
     text-align: right;
+
+    svg {
```

Comment:
Why this change is needed?

Reply:
This targets only the svg on the left side of the slider and makes it flush against the slider. Without this change, the slider looks like this: 
![image](https://user-images.githubusercontent.com/37936169/181621677-85278970-673a-4a95-b1d3-b5b9f70292c0.png)
vs the correct: 
![image](https://user-images.githubusercontent.com/37936169/181621731-9f56bb1b-0e1b-49ee-8a4c-0c92f91a2665.png)

This is needed because of the change to include `display: flex` in the icon size.